### PR TITLE
Converted project to requirejs

### PR DESCRIPTION
**Pull request by sumnerevans**

### CODE REVIEW

File: server/controller/root.py [hunk 1]
```diff
@@ -112,13 +112,17 @@ def new_login():
     return {'page': 'new_login'}
 
 
-@route('/resources/<filepath:path>')
-def static(filepath):
+@route('/<resource_root>/<filepath:path>')
+@route('/<resource_root>/<filepath:path>')
```
Comment (daichij) [inline server/controller/root.py]: Why are there two of the exact same root?

File: server/js/app/device-metric-graphs.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 /**
-* @fileoverview Contains code for graphing device metrics using D3
-*/
+ * @fileoverview Contains code for graphing device metrics using D3
```
Comment (daichij) [inline server/js/app/device-metric-graphs.js]: Are we using D3 at all anymore? I thought we were using chart.js

File: server/js/app/wireless-debug.js [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+   */
+  start() {
+    this.ws_ = new WebSocket(`ws://${location.host}/ws`);
+    this.ws_.onopen = () => this.websocketOnOpen();
+    this.ws_.onmessage = (message) => this.websocketOnMessage(message);
+
+    let api_key = $('#api-key');
+    let device = $('#device');
+    let app = $('#app');
+    let time = $('#start-time');
```
Comment (TiffanyKalin) [inline server/js/app/wireless-debug.js]: Add here:
    `let data_table = $('#historical-log-table').DataTable();
    data_table.destroy();`

File: server/js/app/wireless-debug.js [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    let data_table = $('#historical-log-table').DataTable();
+
+    device.on('change', () => {
+      let chosen_device = device.val();
+      // Gets rid of old data but keeps table structure.
+      data_table.destroy();
```
Comment (TiffanyKalin) [inline server/js/app/wireless-debug.js]: Need to add `data_table = $('#historical-log-table').DataTable();` before the `data_table.destroy()` This keeps the table structure and only allows for a DataTable when the session is started.

File: server/js/app/wireless-debug.js [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+        $('#hidden-dev-alias').css('display', 'none');
+        $('#hidden-app').css('display', 'none');
+        $('#hidden-start').css('display', 'none');
+        $('#historical-log-table tbody tr').remove();
+      }
+    });
+    app.on('change', () => {
+      let chosen_app = app.val();
+      // Gets rid of old data but keeps table structure.
+      data_table.destroy();
```
Comment (TiffanyKalin) [inline server/js/app/wireless-debug.js]: Need to add `data_table = $('#historical-log-table').DataTable();` before the `data_table.destroy()` This keeps the table structure and only allows for a DataTable when the session is started.

File: server/js/app/wireless-debug.js [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+        $('#hidden-app-alias').css('display', 'none');
+        $('#hidden-start').css('display', 'none');
+        $('#historical-log-table tbody tr').remove();
+      }
+    });
+
+    time.on('change', () => {
+      let chosen_starttime = time.val();
+      // Gets rid of old data but keeps table structure.
+      data_table.destroy();
```
Comment (TiffanyKalin) [inline server/js/app/wireless-debug.js]: Need to add `data_table = $('#historical-log-table').DataTable();` before the `data_table.destroy()` This keeps the table structure and only allows for a DataTable when the session is started.

File: server/js/app/wireless-debug.js (hunk 4 above)
Comment (sumnerevans) [inline server/js/app/wireless-debug.js]: let data_table = $('#historical-log-table').DataTable();

is moved to line 42.

File: server/controller/root.py [hunk 7]
```diff
@@ -112,13 +112,16 @@ def new_login():
     return {'page': 'new_login'}
 
 
-@route('/resources/<filepath:path>')
-def static(filepath):
+@route('/<resource_root>/<filepath:path>')
+def static(resource_root, filepath):
```
Comment (sumnerevans) [inline server/controller/root.py]: https://github.com/sumnerevans/wireless-debugging/commit/bfe31ff985b31334b79c8b079eb14ce37dea1b3e

File: server/js/app/wireless-debug.js (hunk 4 above)
Comment (TiffanyKalin) [inline server/js/app/wireless-debug.js]: It has to be elsewhere also. In all the places indicated.

**Pull request merged**